content/en/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,151 @@
+---
+reviewers:
+- zparnold
+title: Overview of Cloud Native Security
+content_template: templates/concept
+weight: 1
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+Kubernetes Security (and security in general) is an immense topic that has many
+highly interrelated parts. In today's era where open source software is
+integrated into many of the systems that help web applications run,
+we need some overarching concepts to help guide our intuition about how we can
```

Comment:
I think `open source software ... web applications run` is too general

Reply:
That's fair, what would you suggest?

Reply:
I think you need to be more explicit about what is k8s security? I think the `open source ...` paragraph is not that useful in defining the `security thought framework`.